docs/reference/ois/next/processing.md

```diff
@@ -77,14 +94,89 @@ post-processing.
+
+The processing schema is the same for both
+[`preProcessingSpecificationV2`](/reference/ois/next/specification.md#_5-11-preprocessingspecificationv2)
+and
+[`postProcessingSpecificationV2`](/reference/ois/next/specification.md#_5-12-postprocessingspecificationv2).
+Snippets for both specifications follow this schema:
+
+- `environment` - Currently only possible value is `Node`. This options
+  interprets the code as JavaScript and executes it in Node.js. The function can
+  be also asynchronous (async/await is supported as well). The processing
+  implementation will wait for the function to resolve.
+- `value` - The processing code written as a string.
+- `timeoutMs` - The maximum timeout that this snippet can run. In case the
+  timeout is exceeded an error is thrown.
+
+Try the [Post processing](/guides/airnode/post-processing/) guide to further
+understand pre/post processing.
+
+#### Input and Output
+
+The processing snippet is a function which receives payload as an argument. The
+return value of the function is treated as a processing result. Apart from the
+payload argument, you can use most Node.js built-in modules.
+
+The payload argument for pre-processing is an object with the following
+property:
+
+- `endpointParameters` - The endpoint parameters with the exception of reserved
+  parameters. For example, if there was a parameter named `myParameter` defined
+  in the `endpoints[n].parameters` array, its value could be accessed using
+  `endpointParameters.myParameter` within pre-processing snippet.
+
+The output of the pre-processing snippet is an object with the following
+property:
+
+- `endpointParameters` - The pre-processed endpoint parameters parameters. These
+  are used to make the API call.
+
+The payload argument for post-processing is an object with the following
+properties:
+
+- `response` - The response of the underlying data provider API call. In case of
+  Airnode skipping the API call, the `response` contains the output of
+  pre-processing snippet.
+- `endpointParameters` - The raw (not pre-processed) endpoint parameters with
+  the exception of reserved parameters. For example, if there was a parameter
+  named `myParameter` defined in the `endpoints[n].parameters` array, its value
+  could be accessed using `endpointParameters.myParameter` within pre-processing
+  snippet.
+
+The output of the post-processing snippet is an object with the following
+properties:
+
+- `response` - The post-processed API call response (or post-processed result of
+  pre-processing snippet in case of skipping an API call). This is used to
+  encode the response values defined by reserved parameters.
+- `timestamp` - (Optional) The timestamp of the API call response. Use this if
+  you want Airnode to use a specific timestamp (instead of a current time at
+  request processing) when using the [signed data
+  gateway](https://docs.api3.org/reference/airnode/latest/understand/http-gateways.html#http-signed-data-gateway).
```

Comment:
Local link:
```suggestion
  gateway](/reference/airnode/latest/understand/http-gateways.md#http-signed-data-gateway).
```

Reply:
Thanks. I also double checked that the link works on the preview site (https://vitepress-docs--pr669-processing-v2-d3oh59pa.web.app/reference/airnode/latest/understand/http-gateways.html#http-signed-data-gateway).